### DISCUSSION TITLE
Writing force file from arc file

### DISCUSSION THREAD

**Discussion by jacekkozuch** [TinkerTools/tinker#112] [Q&A] [answered]
Dear all,

I have seen that there's the option "save-forces" similar to "save-induced". However, till now I've only found how to print the ____.uind file after having already performed the MD production run (i.e. using the analyze option). Is there a way how to write the ____.force file in a similar way from and existing arc file?
If not, do you see an easy way how change a few lines in the analyze code (introduce another option for instance), so that this can be done?

Thank you in advance!

Best
Jacek

**Comment by jayponder** [comment]
If you include the "SAVE-INDUCED" keyword with Tinker (a.k.a. "Tinker8", the CPU version), then a file *.uind with the induced dipole components on each atom will be written during an MD simulation and alongside the *.arc trajectory file.

Similarly, if you include the "SAVE-FORCE" keyword, a *.frc file will be written with the force components on each atom.

I believe (not 100% sure) that both of these keyword options are only available in CPU Tinker8, and are ignored by the GPU Tinker9 code. In order to maintain efficiency, many of these kinds of options that are available in Tinker8 are not included in the GPU version.

**Reply by jacekkozuch** [reply, marked as answer]
Just providing the link to the TINKER9 Discussion, where this was solved:

https://github.com/TinkerTools/tinker9/discussions/200
